op-service/sources/types.go

```diff
@@ -29,6 +31,78 @@ import (
 //
 // This way we minimize RPC calls, enable batching, and can choose to verify what the RPC gives us.
 
+// headerInfo is a conversion type of types.Header turning it into a
```

Comment:
not sure if this diff was supposed to make it in

Reply:
It's not. We de-duplicated this type in develop during the onsite. We shouldn't bring back a duplicate.

Reply:
ok, will delete, thank you

Reply:
deleted